### ISSUE TITLE
Experiment with stateful vs stateless

### ISSUE THREAD

**Issue by stefanwalther** [stefanwalther/qix-graphql#13]
It would be interesting to run the GraphQL server in two different modes (based on configuration):

- **Stateful**
  - ws connection remains open
  - user can make selections
  - can get current selections
  - can execute a chain of commands
  - etc.
- **Stateless**
  - always opening, closing, re-opening the ws-connection
  - do not allow the user to make selections
  - some mutations and queries should then just not be available
  - **Basically the GraphQL server acts then like a connector to fetch data easily from a Qlik app**
